doc/varnish-configuration.rst

```diff
@@ -33,6 +33,18 @@ simple ACL, valid for all Varnish versions from 3 onwards, looks as follows:
     trigger invalidation are whitelisted here. Otherwise, lost cache invalidation
     requests will lead to lots of confusion.
 
+Advance Varnish Configuration: xkey
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With Varnish 4 and 5 you can optionally take advantage of official Varnish
+modules, specifically `xkey Varnish module`_. This module will provide better
+performance for tag handling as it adds secondary hashes to objects, and as
+it allows for grace handling by means of softpurge on tags.
```

Comment:
can we say what it does and then just (softpurge) in paranthesis? i am not sure if people know well enough what softpurge means.